src/Microsoft.OData.Core/ODataBatchOperationRequestMessage.cs

```diff
@@ -92,6 +103,14 @@ public string Method
         }
 
         /// <summary>
+        /// Gets the requst prerequisites.
+        /// </summary>
+        public IList<string> DependsOnRequestIds

```

Comment:
I believe your thought is from the "DependsOn" request attribute in the Spec. The "DependsOn" in the Spec is a list of request Ids and group Ids. However, the member variable here only contains "RequestIds", considering groupIds are not the same between request and response in current protocol (for both multipart and json formats) so the only reliable way for the client to ensure that prerequisites of a request is satisfy is thru request ids.